tests/installation/system_role/validate_default_role_gnome_desktop.pm

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2021 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# Summary: Test module to validate default System Role during installation
```

Comment:
we can delete 'default' from the name of all tests, so we can use this test module to validate default or validate what we selected, so `validate_role_gnome_desktop` should be enough.

Reply:
I like the idea to shorten the names and use them to validate after, but then it will be hard to understand why this module preceeds the module selection.

Reply:
then we can use `validate_selected_role_gnome_desktop`, 'selected' would work for both, wdyt?
(I think we discuss something like this a lot of time ago in bootloader treating defaults or pre-selected values)

Reply:
That would do, good idea !